### ISSUE TITLE
Add a global --app option to allow accessing an app outside its directory

### ISSUE THREAD

**Issue by wilzbach** [storyscript/cli#144]
Currently there's no way to access an app outside its directory.

```
story --app <foo> url
```

```
story --app logs
```

(instead of the individual `story logs --app ...`)

**Comment by judepereira** [comment]
--app was present, but probably didn't work with the old logs command. It works with the new one.

**Comment by judepereira** [comment]
Also, the url works too.

**Comment by judepereira** [comment]
@wilzbach no?

**Comment by wilzbach** [comment]
> --app was present, but probably didn't work with the old logs command. It works with the new one.

Awesome, but this issue is about a global --app, not a semi-global --app ;-)
The reason for this is that it's much more intuitive for the user and also convenient for aliases.
So the following should work:

```
story --app <foo> apps url
```

In fact, with `--app` given, it's clear that a specific app is selected so it could be:

```
story --app <foo> url
```

**Comment by judepereira** [comment]
```
$ story apps url --app login
https://login.storyscriptapp.com/
```
Every command supports it, except for the deploy command.

**Comment by judepereira** [comment]
```
$ story deploy --app login
The --app option is not allowed with the deploy command.
```

**Comment by wilzbach** [comment]
> Awesome, but this issue is about a global --app, not a semi-global --app 

`--app login` needs to be able to appear at _front_ (the first parameter), because otherwise one can't create convenient aliases. As mentioned earlier other CLIs like e.g. Heroku have the same behavior for exactly that reason.

Anyhow, with click it's not that hard, it's just adding a `click.option` to the main CLI here:

https://github.com/storyscript/cli/blob/97360a9bc84d2a96c809726a2ba3c41fe44cae2d/story/cli.py#L352-L353

**Comment by kenneth-reitz** [comment]
I agree, that it should simply be a global. That'll open the arg up to environment variable scripting, too.
